packages/react-dom/src/server/ReactPartialRendererHooks.js

```diff
@@ -474,6 +474,13 @@ function useTransition(
   return [startTransition, false];
 }
 
+function useEvent(event: any): any {
```

Comment:
Same question for this. Why is the return type `any` rather than `ReactListenerMap`?